src/lib/isNumeric.js

```diff
@@ -5,8 +5,21 @@ const numericNoSymbols = /^[0-9]+$/;
 
 export default function isNumeric(str, options) {
   assertString(str);
+
   if (options && options.no_symbols) {
     return numericNoSymbols.test(str);
   }
-  return (new RegExp(`^[+-]?([0-9]*[${(options || {}).locale ? decimal[options.locale] : '.'}])?[0-9]+$`)).test(str);
+
+  const decimal_char = (options || {}).locale ? decimal[options.locale] : '.';
+  if (options && options.thousands_separator) {
+    const separator = `${options.thousands_separator || ''}`;
+    if (separator.length > 1 || numericNoSymbols.test(separator)) {
```

Comment:
Also, since we just need to do space or comma, let's validate based on the two characters only for validaty.

Reply:
@ezkemboi 
From what I read in [wikipedia](https://en.wikipedia.org/wiki/Decimal_separator), the decimal separator can also be a period(.) other than a comma(,) or space( ). 
In some programming languages, they also use an underline separator(_) for readability. 

That's why I think it would be easier to use if we make it customizable.